docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc

```diff
@@ -23,10 +23,10 @@ This collector provides tools for parsing and mapping the metrics to the collect
 |===
 | Parameter       | Description                                    | Required | Default value
 | `collection`    | The name of the _Prometheus Collection_ to use | required |
-| `url`           | HTTP URL to query for the metrics              | required |
+| `url`           | HTTP URL to query for the metrics (most likely will be `http://INTERFACE_ADDRESS:9100/metrics`)  | required |
```

Comment:
Should we consider using the meta-data syntax i.e. `${interface:address}` here in the example instead `INTERFACE_ADDRESS`?

Reply:
given https://issues.opennms.org/browse/NMS-13257 I kept PSM docs consistent.

Reply:
Yeah, it doesn't work everywhere, but it does work more often than `INTERFACE_ADDRESS` does.

I'm good either way, just caught my attention :)

Reply:
Also: https://github.com/OpenNMS/opennms/blob/7e9f5759badaf5636ae487a72b605a2a89e3e13e/features/prometheus-collector/src/main/java/org/opennms/netmgt/collectd/prometheus/PrometheusCollector.java#L99

Reply:
In a test, `${interface:address}` gives me a "bad substitution" error in Karaf while `INTERFACE_ADDRESS` returns data.  I'll open a JIRA as it doesn't look like meta data is properly supported on this collector.

Reply:
you have to quote the value of the url parameter.  `url='http://${interface:address}:9182/metrics'` works.

Reply:
After realizing I had other errors in my Prometheus config, I was able to get collection using the meta-data syntax in my collectd service.  I haven't had a case where I've needed to collect from Prometheus, though I was curious if there is any reason to have a default value for the URL if that parameter is omitted. I don't know how common it is for people to override the port or `/metrics` path.